app/views/posts/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<h3>Message Board</h3>
+
+<% if user_signed_in? %>
+  <%= render 'form', post: @post %>
+<% end %>
+
+<% @posts.each do |post| %>
+  <div class="card" style="width: 100%;">
+    <div class="card-body">
+      <% if user_signed_in? %>
+        <h5 class="card-title">@<%= post.user.username %></h5>
+      <% end %>
+
+      <p class="card-text"><%= post.body %></p>
+      <% if user_signed_in? && post.user_id == current_user.id %>
+        <%= link_to 'Edit', edit_post_path(post), class: 'card-link' %>
+        <%= link_to 'Destroy', post, method: :delete, data: { confirm: 'Are you sure?' }, class: 'card-link' %>
+      <% end %>
+    </div>
+  </div>
+<% end %>
```

Comment:
* [ ] **_There should be NO application logic in the views (no `Post.new`, `Comment.create`, `current_user.friends.find_by(...)`...). Everything connected with data processing should be encapsulated in models and used in controllers._**
* [ ] **_View-related logic can be encapsulated in View helpers (app/helpers/\*_helper.rb)_**
* [ ] **_There should be absolutely NO logic in views._**